src/txn/txn_ckpt.c

```diff
@@ -546,6 +546,7 @@ __checkpoint_prepare(WT_SESSION_IMPL *session, bool *trackingp, const char *cfg[
     __wt_epoch(session, &conn->ckpt_prep_start);
 
     WT_RET(__wt_txn_begin(session, txn_cfg));
+    __checkpoint_timing_stress(session, WT_TIMING_STRESS_PREPARE_CHECKPOINT_DELAY, 0, 1000);
```

Comment:
Is it possible to use constant(e.g., PREPARE_STRESS_DELAY_DURATION) instead of using some magic number? Same comment for other similar calls to `__checkpoint_timing_stress()`.

Reply:
Maybe a comment would suffice?

Reply:
Changing the function parameter `struct timespec` let me use an existing constant here and I added a comment to the other calls.